site/_posts/2017-03-12-marching-ahead-embedded-ansible.md

```diff
@@ -0,0 +1,81 @@
+as added by [Mooli](https://github.com/moolitayer) [here](https://github.com/ManageIQ/manageiq/pull/12773).
+
+In the Service UI, RBAC is now enabled on [Orchestration Templates](https://github.com/ManageIQ/manageiq-ui-service/pull/576)
+
+The classic UI toolbar refactoring is moving along with lots of PRs merged, the full list can be found [here](https://github.com/ManageIQ/manageiq/issues/6259).
+
+
+## Fixed
+
+[JoeR](https://github.com/jrafanie) fixed a failure that would occur after starting new puma workers and try to gracefully restart apache.
+In his [PR](https://github.com/ManageIQ/manageiq/pull/14007), Apache is configured to boot the ports puma workers are bound to (from STARTING_PORT to the maximum worker count 
+port 3000 to 3009 if max worker count is 10) as balancer members.
+
+[Libor](https://github.com/lpichler) created a fix in this [PR](https://github.com/ManageIQ/manageiq/pull/14229) to allow regex for a
+MiqExpression::Field containing numbers in associations.
+
+## New
+
+[Rails 5.0.2.](https://github.com/ManageIQ/manageiq/pull/14175) is now supported.
+
+Check out [Ladas'](https://github.com/ladas) Target and Target Collections abstractions [here](https://github.com/ManageIQ/manageiq/pull/14249).
+
+The Embedded Ansible model is now being used when retrieving [credentials](https://github.com/ManageIQ/manageiq-ui-classic/pull/627) 
+or [repositories](https://github.com/ManageIQ/manageiq-ui-classic/pull/659) for catalog items, as well as [list pages](https://github.com/ManageIQ/manageiq-ui-classic/pull/584).
+
+With this new [PR](https://github.com/ManageIQ/manageiq/pull/14225) from [Bill](https://github.com/bzwei) - the Ansible Play data, including start time and finish time, is collected during a Job run. 
+The credentials are associated with the job during a [sync](https://github.com/ManageIQ/manageiq/pull/14113).
+
+Associated resources can now be removed as part of the service retirement, with [this UI option](https://github.com/ManageIQ/manageiq-ui-classic/pull/629) added by [Harpreet]( https://github.com/h-kataria). 
+
+The UI for the physical infrastructure is now available, added by [Julian](https://github.com/juliancheal) in this [PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/196). 
+
+For the REST API, [Daniel](https://github.com/dkorn) added [REST API](https://github.com/ManageIQ/manageiq/pull/13967) support for Alert Definitions (Miq Alert), while [Jillian](https://github.com/jntullo) 
+added the read API for [configuration scripts](https://github.com/ManageIQ/manageiq/pull/14223) and access to the jobs on a [ServiceAnsiblePlaybook](https://github.com/ManageIQ/manageiq/pull/14248).
+
+
+## Deleted
+
+[Martin](https://github.com/martinpovolny) uses GenericShowMixin to unify and remove #show functionality from middeware controllers -[PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/590)
+
```

Comment:
@lgalis - Little late to the party - possibly add a space between the '-' and PR?